.travis.yml

```diff
@@ -9,11 +9,17 @@ env:
 services:
   - postgresql
   - redis-server
+  - docker
 install:
   - pip install -r requirements.txt
 script:
   - flake8
   - python manage.py test
+  - docker build -t pyslackers/website:$TRAVIS_BRANCH .
 matrix:
   allow_failures:
-    - "3.7-dev"
+    python: "3.7-dev"
```

Comment:
Lists aren't a valid item under `allow_failures`, this was reported by the travis linter